.github/workflows/haskell-ci.yml

```diff
@@ -159,6 +159,12 @@ jobs:
           rm -f cabal-plan.xz
           chmod a+x $HOME/.cabal/bin/cabal-plan
           cabal-plan --version
+      - name: install hlint
```

Comment:
Is haskell-ci smart enough to run hlint in the ghc 9.0.2 run without additional flags?

Reply:
I don't understand what you mean, sorry. I did not look at this file after generating it.

Reply:
How did you generate it? Did you have to provide a "use ghc902 for the hlint step"?

Reply:
No. I ran `haskell-ci --hlint-version '>=3.3 && <3.5' regenerate`. The next release of haskell-ci will allow hlint 3.4 by default so won't be needed.